src/domain/Statuses/Status/Status.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+
+import { Props, Variables } from '../../../common'
+import { Text } from '../../Typographies/Text'
+import { StatusDot, StatusDotVariants } from '../StatusDot'
+import { StyledStatus } from './style'
+
+enum Size {
+  Small = 'small',
+  Large = 'large'
```

Comment:
should this be normal/regular/default/medium? whatever the word we use for that size of typography